youtubesearchpython/__future__/streamurlfetcher.py

```diff
@@ -52,10 +52,9 @@ async def get(self, videoFormats: dict, itag: int) -> Union[str, None]:
             >>> print(url)
             "https://r6---sn-gwpa-5bgk.googlevideo.com/videoplayback?expire=1610798125&ei=zX8CYITXEIGKz7sP9MWL0AE&ip=2409%3A4053%3A803%3A2b22%3Adc68%3Adfb9%3Aa676%3A26a3&id=o-APBakKSE2_eMDMegtCmeWXfuhhUfAzJTmOCWj4lkEjAM&itag=251&source=youtube&requiressl=yes&mh=aP&mm=31%2C29&mn=sn-gwpa-5bgk%2Csn-gwpa-qxad&ms=au%2Crdu&mv=m&mvi=6&pl=36&initcwndbps=146250&vprv=1&mime=audio%2Fwebm&ns=ULL4mkMO31KDtEhOjkOrmpkF&gir=yes&clen=10210834&dur=634.601&lmt=1544629945422176&mt=1610776131&fvip=6&keepalive=yes&c=WEB&txp=5511222&n=uEjSqtzBZaJyVn&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cns%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRAIgKKIEiwQTgXsdKPEyOckgVPs_LMH6KJoeaYmZic_lelECIHXHs1ZnSP5mgtpffNlIMJM3DhxcvDbA-4udFFE6AmVP&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhAPmhL745RYeL_ffgUJk_xJLC-8riXKMylLTLA_pITYWWAiB2qUIXur8ThW7cLfQ73mIVK61mMZc2ncK6FZWjUHGcUw%3D%3D"
         '''
-        self._getDecipheredURLs(videoFormats)
-        for stream in self._streams:
-            if stream["itag"] == itag:
-                return stream["url"]
+        self._getDecipheredURLs(videoFormats, itag)
+        if len(self._streams) == 1:
```

Comment:
Fails when I try to use fetcher multiple times:
```python
from youtubesearchpython import Video, StreamURLFetcher

fetcher = StreamURLFetcher()
video = Video.get("https://www.youtube.com/watch?v=cinJDxLUsNY")

print(1, fetcher.get(video, 251) != None)
print(2, fetcher.get(video, 251) != None)
```
```
1 True
2 False
```

Reply:
It should work now